<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiologistFormClass</class>
 <widget class="QMainWindow" name="BiologistFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiologistForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBoxSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewBacterium">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Add bacterium</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Species</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSpecies"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditSize"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Diseases:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditDiseases"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonAddBacterium">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetDiseases">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>221</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddDisease">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add disease</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BiologistForm.qrc"/>
 </resources>
 <connections/>
</ui>
